core/src/main/java/io/fabric8/maven/core/util/ProcessUtil.java

```diff
@@ -123,6 +127,14 @@ protected static void processOutput(Logger log, InputStream inputStream, Functio
     public static File findExecutable(Logger log, String name) {
         List<File> pathDirectories = getPathDirectories(log);
         for (File directory : pathDirectories) {
+            if( isWindows() ) {
```

Comment:
you're right

seems that the best would be to be portable and use a java library